src/app/frontend/deploy/deployfromsettings_controller.js

```diff
@@ -347,10 +347,19 @@ export default class DeployFromSettingsController {
    */
   getContainerImageVersion_() {
     /** @type {number} */
-    let index = (this.containerImage || '').lastIndexOf(':');
+    let index = (this.containerImage || '').lastIndexOf('/');
```

Comment:
Or am I wrong? Can you convince me and other code readers that this algorithm works?